.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "m2e-core-tests"]
 	path = m2e-core-tests
-	url = https://github.com/tesla/m2e-core-tests.git
-	branch = master
+	url = https://github.com/HannesWell/m2e-core-tests.git
```

Comment:
Has to be reverted before this can be merged.
Requires https://github.com/tesla/m2e-core-tests/pull/185

Reply:
Done.